rskj-core/src/test/java/org/ethereum/config/blockchain/upgrades/ActivationConfigTest.java

```diff
@@ -60,6 +61,7 @@
             "    rskip126: wasabi100",
             "    rskip132: wasabi100",
             "    rskip136: bahamas",
+            "    rskip140: wasabi120",
```

Comment:
Besides the _wasabi120_ reference, I don't think this test needs to be modified on every new _network upgrade_, I think it's testing the features avaiable through the configuration mechanism, not and exhaustive existence of every _network upgrade_ definition.

Reply:
The test should be modified to make it pass